src/audio/host-zephyr.c

```diff
@@ -393,6 +393,30 @@ static int host_get_status(struct comp_dev *dev, struct host_data *hd, struct dm
 /* Minimum time between 2 consecutive "no bytes to copy" messages in milliseconds */
 #define SOF_MIN_NO_BYTES_INTERVAL_MS 20
 
+static inline bool host_handle_eos(struct host_data *hd, struct comp_dev *dev,
```

Comment:
can this be more meaningful name as to what this function does, it looks like its checking for EOS and returning true if EOS.  e.g. would be more readable as```is_stream_eos()```